idm/pkg/metrics/metrics.go

```diff
@@ -0,0 +1,45 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	// Namespace defines the namespace for the defines metrics.
+	Namespace = "ocis"
+
+	// Subsystem defines the subsystem for the defines metrics.
+	Subsystem = "idm"
+)
+
+// Metrics defines the available metrics of this service.
+type Metrics struct {
+	// Counter  *prometheus.CounterVec
+	BuildInfo *prometheus.GaugeVec
+}
+
+// New initializes the available metrics.
+func New() *Metrics {
+	m := &Metrics{
+		// Counter: prometheus.NewCounterVec(prometheus.CounterOpts{
+		// 	Namespace: Namespace,
+		// 	Subsystem: Subsystem,
+		// 	Name:      "greet_total",
+		// 	Help:      "How many greeting requests processed",
+		// }, []string{}),
```

Comment:
Can we remove the commented code or should we uncomment it? (`Counter` is also commented in the `Metrics` struct)

Reply:
I'll remove it for now. Still need to figure out how we can integrate the metrics that libreraph/idm provides. Into the ocis service.

Reply:
I've removed the monitoring and tracing pieces for now. They were just stubs anyway